helm-chart-sources/mysql-exporter/Chart.yaml

```diff
@@ -7,5 +7,7 @@ appVersion: v0.12.1
 sources:
   - https://github.com/prometheus/mysqld_exporter
 maintainers:
-  - name: thangaraj ravi
-    email: thangaraj.ravi@bluescape.com
+  - name: juanchimienti
+    email: juan.chimienti@gmail.com
+  - name: monotek
+    email: monotek23@gmail.com
```

Comment:
I know this is a bit pedantic and annoying, but I'd put your information in as a maintainer as well, which will allow us to know you are the local maintainer and anyone (like infra) knows they should come to you to ask questions. Alternatively, you could put the SRE team down as a maintainer.

Reply:
Sure @notjames , Will do that.

Reply:
Done @notjames